modules/utxo-lib/src/bitgo/wallet/Unspent.ts

```diff
@@ -186,11 +188,12 @@ export function updateReplayProtectionUnspentToPsbt(
   }
 }
 
-function addUnspentToPsbt(psbt: UtxoPsbt, id: string): void {
+function addUnspentToPsbt(psbt: UtxoPsbt, id: string, sequenceNumber: number = DEFAULT_TX_INPUT_SEQUENCE_NUMBER): void {
```

Comment:
```suggestion
function addUnspentToPsbt(psbt: UtxoPsbt, id: string, { sequenceNumber  = DEFAULT_TX_INPUT_SEQUENCE_NUMBER }: { sequenceNumber?: number } = {}): void {
```

more extensible like this